.idea/gradle.xml

```diff
@@ -4,9 +4,11 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
+        <option name="delegatedBuild" value="true" />
+        <option name="testRunner" value="GRADLE" />
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="12" />
+        <option name="gradleJvm" value="13" />
```

Comment:
Is this the new standard Java version that everyone should be using?

Reply:
I think WPILib actually recommends Java 11 instead of either 12 or 13. It is set to Java 12 on master, so I'll update it to 11